Sources/ProjectDescription/Target.swift

```diff
@@ -13,7 +13,7 @@ public struct Target: Codable, Equatable {
     public let product: Product
 
     /// Bundle identifier.
-    public let bundleId: String
```

Comment:
I am wondering, how does it work if we don't specify it anywhere as it being now turned into `optional`?

Reply:
Anything that you have in a ‘Target’ can be set in ‘xcconfig’ file. That’s why it make sense add ’Target(xcconfig:)’ instead of current implementation.